sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -153,6 +153,7 @@ case class Like(left: Expression, right: Expression) extends StringRegexExpressi
   }
 }
 
+// scalastyle:off line.contains.tab
```

Comment:
We need to check tab separators in examples when testing them (I mean we cannot ignore tabs in tests)? I feel a little annoying to add this line...

Reply:
Do you want to modify output of examples and replace tabs by something else? Just in case, we have to do the same while writing new examples, right. So, you cannot just copy-paste output from a terminal as is. If someone doesn't know this, he/she will lose time on troubleshooting test failures.